src/Migrator.php

```diff
@@ -129,6 +144,23 @@ protected function runMigrations(array $config): void
         }
     }
 
+    /**
+     * Truncates connections on demand.
+     *
+     * @param string[] $connections Connections names to truncate. Defaults to modified connections
+     * @return void
+     */
+    public function truncate(array $connections = []): void
```

Comment:
I would rather put an `?array = null` as parameter, and assign `$this->connectionsWithModifiedStatus` to `$connections` only if null is passed as argument.
The reason is that `$connections` being an empty array should mean IMO that no connection should be truncated, while passing no arguments does mean truncate all.